CMakeLists.txt

```diff
@@ -284,6 +287,11 @@ if (GGML_CUDA_SOURCES)
     set_property(TARGET rwkv PROPERTY CUDA_ARCHITECTURES OFF)
 endif()
 
-enable_testing()
-add_subdirectory(tests)
-add_subdirectory(extras)
+if (NOT RWKV_STANDALONE)
+    set_property(TARGET ggml PROPERTY GGML_STANDALONE OFF)
+    enable_testing()
+    add_subdirectory(tests)
+    add_subdirectory(extras)
+elseif()
+    set_property(TARGET ggml PROPERTY GGML_STANDALONE ON)
+endif()
```

Comment:
Please add new line at the end of the file

Reply:
Should be fine now...